packages/styles/gulp/tasks/ltr-sass.js

```diff
@@ -49,6 +62,6 @@ function _sass() {
 /**
  * Gulp task export
  *
- * @module sass
+ * @module sass-ltr
  */
-module.exports = gulp.task('sass', _sass);
+module.exports = gulp.task('sass-ltr', ltr);
```

Comment:
@IgnacioBecerra super nitpick, can you name the files to match the task name? `sass-ltr.js` and `sass-rtl.js`.

Reply:
Done!